pom.xml

```diff
@@ -171,12 +177,5 @@
         <version>1.11-R0.1-SNAPSHOT</version>
         <scope>provided</scope>
     </dependency>
-    <!--Bukkit API-->
```

Comment:
The Spigot API includes the Bukkit API. No Spigot-specific features are used, which would be preferred?